src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClientX/SqlConnectionX.cs

```diff
@@ -85,13 +150,46 @@ public override Task ChangeDatabaseAsync(string databaseName, CancellationToken
         public object Clone()
             => throw new NotImplementedException();
 
+        #region close
+
         /// <inheritdoc/>
         public override void Close()
-            => throw new NotImplementedException();
+            => Close(async: false).GetAwaiter().GetResult();
```

Comment:
Is this equivalent to `Close(async: false).Wait()`?

Reply:
From what I can tell, the main difference is that this preserves any exceptions and Wait() will give an AggregateException.
https://blog.stephencleary.com/2014/12/a-tour-of-task-part-6-results.html